### PR TITLE
[WEB-9][WEB-10] Allow users to change group settings

### PR DESCRIPTION
**Pull request by trezy**
See [WEB-9](https://rollforguild.atlassian.net/browse/WEB-9) and [WEB-10](https://rollforguild.atlassian.net/browse/WEB-10) for details.

### CODE REVIEW

File: components/GroupProfilePanels/GroupSettingsPanel.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+              name="name"
+              onChange={this._handleChange}
+              pattern="[\w\s_-]+"
+              placeholder="e.g. Quigley's Tavern"
+              required
+              type="text"
+              value={name} />
+          </fieldset>
+
+          {/* <fieldset>
```
Comment (UncleClapton) [inline components/GroupProfilePanels/GroupSettingsPanel.js]: Was this intentionally left commented out?

File: components/GroupProfilePanels/GroupSettingsPanel.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+            <AddressInput
+              disabled={submitting}
+              id="address"
+              onChange={this._handleAddressChange}
+              required
+              value={address} />
+          </fieldset>
+
+          {/* <fieldset className="horizontal">
```
Comment (UncleClapton) [inline components/GroupProfilePanels/GroupSettingsPanel.js]: I take it we're still waiting on the API fix for this?

File: components/GroupProfilePanels/GroupSettingsPanel.js (hunk 1 above)
Comment (trezy) [reply components/GroupProfilePanels/GroupSettingsPanel.js]: It was. The API doesn't support changing the permalink yet. When it does, we can just uncomment this component.

File: components/GroupProfilePanels/GroupSettingsPanel.js (hunk 2 above)
Comment (trezy) [reply components/GroupProfilePanels/GroupSettingsPanel.js]: Yup, discoverability still doesn't come back from the API.

**Pull request merged**